### ISSUE TITLE
Unknown provider: copilot

### ISSUE THREAD

**Issue by VSCodeTriageBot** [microsoft/vscode#205592]
```javascript
Error: Unknown provider: copilot
at P.U in src/vs/workbench/contrib/chat/common/chatServiceImpl.ts:369:11
```
[Go to Errors Site](https://errors.code.visualstudio.com/card?ch=00124e9e5830e3efc897db71c781899f8a676295&bH=e26af844-27a9-de9b-0fa7-9e9e650006e5)

**Comment by vitaliy-guliy** [comment]
@roblourens is there any news on this?
We are experimenting with the extension in downstream Che-Code project.
But to be honest, since copilot chat is not open source, it is difficult to understand what's happening on the extension side.

**Comment by roblourens** [comment]
Fixed in #207128

@vitaliy-guliy that would imply that the Copilot Chat extension failed to activate or register its provider for some reason
